### ISSUE TITLE
plot performance like ;plot rating

### ISSUE THREAD

**Issue by krofna** [cheran-senthil/TLE#47]

**Comment by krofna** [comment]
precalculate performances for all users in all contests and store in cache.
see #43 #196

**Comment by Aaerialys** [comment]
this shouldn't be too hard right? just approximate it as current rating + delta*4

**Comment by krofna** [comment]
thats not performance. thats rating + delta*4

**Comment by Aaerialys** [comment]
yes but it is close to the performance for most circumstances and easy to add to the current ;ranklist command because you already know the delta and rating of each person

for example rating+delta*4 is used to calculate performance here: https://greasyfork.org/en/scripts/402180-codeforces-performance

**Comment by krofna** [comment]
I mean actual performance, as calculated by carrot addon.
